data/json/items/melee.json

```diff
@@ -90,7 +93,7 @@
         "color" : "brown",
         "symbol" : "/",
         "material" : ["wood"],
-        "techniques" : ["WBLOCK_1", "RAPID", "PRECISE"],
+        "techniques" : ["WBLOCK_2", "RAPID", "PRECISE"],
```

Comment:
Looks like this json hasn't been formatted properly, so there are tons of format issues (which aren't your fault nor did you add them). So not sure how big a deal of these are unless someone else asks for it to be run through the [linter](http://dev.narc.ro/cataclysm/format.html).

Reply:
Should I wait and make a separate PR that lints melee.json after this is merged or closed, or lint it as part of this PR?

Reply:
Honestly, I'd hazard on the side of linting in a separate PR unless a more senior community member thinks it's better to lint within this PR.

Reply:
Next PR will be better.